### PR TITLE
Add service file for workers with "no cleanup" option

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
A service file that starts a worker service dedicated for debugging test with snapshots.

### CODE REVIEW

**Comment by coolo** [comment]
This is duplicating 99% of the other service file. Use the Makefile to install 2 services from one source - you need to touch the Makefile anyway

**Comment by SergioAtSUSE** [comment]
@coolo, done :)

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,8 @@ install:
 	install -d -m 755 "$(DESTDIR)"/usr/lib/systemd/{system,system-generators}
 	install -d -m 755 "$(DESTDIR)"/usr/lib/tmpfiles.d
 	install -m 644 systemd/openqa-worker@.service "$(DESTDIR)"/usr/lib/systemd/system
+	install -m 644 systemd/openqa-worker@.service "$(DESTDIR)"/usr/lib/systemd/system/openqa-worker-no-cleanup@.service
+	sed -e 's_^\(ExecStart=/usr/share/openqa/script/worker\) \(--instance %i\)$_\1 --no-cleanup \2_' "$(DESTDIR)"/usr/lib/systemd/system/openqa-worker-no-cleanup@.service
```
Comment (coolo) [inline Makefile]: I don't think this does what you want to achieve. sed -e only outputs the changed file to STDOUT.

You either add a -i to the mix - or leave out the install completely and sed -e ... > "$(DESTDIR)"/...

**Comment by SergioAtSUSE** [comment]
@coolo, yeah I forgot to add the "-i". And in one command is also more elegant.

Done

**Comment by coolo** [comment]
Cool. But if I say 'you either ... or ...' I mean you need to pick one and ignore the other - or ignore one and pick the other, not both at the same time :)

**Comment by SergioAtSUSE** [comment]
And so it is. :v: 
The other install is for the normal openqa-worker. I simply split the sed command in multiple lines because it was very long.

**Comment by SergioAtSUSE** [comment]
@coolo, is there something else I should do here?

**Comment by coolo** [comment]
yeah, not sending the output of -i to a file

**Comment by SergioAtSUSE** [comment]
@coolo
ah, that is what you was mentioning with using both forms.
Done.

**Pull request merged**